hack/istio/install-bookinfo-demo.sh

```diff
@@ -140,6 +140,18 @@ else
   exit 1
 fi
 
+IS_OPENSHIFT="false"
+IS_MAISTRA="false"
+if [[ "${CLIENT_EXE}" = *"oc" ]]; then
+  IS_OPENSHIFT="true"
+  if [[ $([ "$(oc get crd | grep servicemesh | wc -l)" -gt "0" ] && echo "true") ]]; then
+    IS_MAISTRA="true"
+  fi
```

Comment:
If you take my recommendation above, remove this `fi` - I forgot to include its removal in my diff above.
```suggestion
```